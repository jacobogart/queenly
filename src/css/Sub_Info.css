/* Testing Purposes */
.Sub-Info {
  /* border: 3px solid orange; */
}
/* ================ */
.Sub-Info {
  background-color: #2e5658;
  border-radius:  0 20px 20px 0;
  padding: 25px;
  box-shadow: #202828 3px 3px 2px;
}