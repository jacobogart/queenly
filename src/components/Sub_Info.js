import React, { Component } from 'react';
// * CSS imports
import '../css/Sub_Info.css';

export default class Sub_Info extends Component {
  constructor(props) {
    super(props);
    this.state = {
    }
  }

  render() {
    return (
      <article className="Sub-Info">
        <p>Sub_Info</p>
      </article>
    )
  }
}