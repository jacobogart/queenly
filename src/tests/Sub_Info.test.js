import React from 'react';
import ReactDOM from 'react-dom';
import Sub_Info from '../components/Sub_Info.js';
import { shallow } from 'enzyme';

const [nameOfFunctionHere] = jest.fn();

describe('Sub_Info', () => {
  let wrapper;
  beforeEach(() => {
    wrapper = shallow(
      <Sub_Info />
    );
  });
  it('should match snapshop', () => {
    expect(wrapper).toMatchSnapshot();
  });
  it('should have default state', () => {
    expect(wrapper.state()).toEqual({
      // Put state here
    });
  });
})