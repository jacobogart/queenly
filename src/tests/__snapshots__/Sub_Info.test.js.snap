// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Sub_Info should match snapshop 1`] = `
<article
  className="Sub-Info"
>
  <div
    className="img-holder"
    dangerouslySetInnerHTML={
      Object {
        "__html": "<iframe src=\\"https://www.google.com/maps/embed?pb=!1m18!1m12!1m3!1d2966.259884708719!2d-87.67001098514663!3d41.97322967921473!2m3!1f0!2f0!3f0!3m2!1i1024!2i768!4f13.1!3m3!1m2!1s0x880fd35871b3d69d%3A0x5fc40651905618a!2sMeeting+House+Tavern!5e0!3m2!1sen!2sus!4v1554176037774!5m2!1sen!2sus\\" width=\\"400\\" height=\\"300\\" frameborder=\\"0\\" style=\\"border:0\\" allowfullscreen></iframe>",
      }
    }
  />
</article>
`;
